pytest.ini

```diff
@@ -47,6 +47,7 @@ markers =
     dfccsd-grad
     dfccsd-t-grad
     dfccsd
+    dfccsdt-grad
```

Comment:
The markers are definitely a mess, mostly because they're a mirror of CTest markers that are also haphazard. A difference is that `ctest -L` matches partial markers but `pytest -m` requires full match. If we do try to reform markers, there's also https://github.com/psi4/psi4/blob/master/psi4/driver/p4util/exceptions.py#L490-L496 as standard transformation. The test names in stdsuite use this.